### PR TITLE
Adding UV Index system channel

### PR DESCRIPTION
**Pull request by clinique**

### CODE REVIEW

**Comment by lolodomo** [comment]
Looks good to me but the build failed. Clicking on "Details" leads to a 404 error. Maybe build should be restarted ?

**Comment by clinique** [comment]
> @clinique for information: build failed in integration tests (thing tests).

Am I supposed to do something ?

**Comment by J-N-K** [comment]
Yes, fix the test 😀

File: bundles/org.openhab.core.thing/src/main/java/org/openhab/core/thing/DefaultSystemChannelTypeProvider.java [hunk 1]
```diff
@@ -397,14 +398,24 @@ public class DefaultSystemChannelTypeProvider implements ChannelTypeProvider {
...
 
+    /**
+     * UV Index: system wide {@link ChannelType} which shows the UV Index
+     */
+    public static final ChannelType SYSTEM_UV_INDEX = ChannelTypeBuilder
+            .state(SYSTEM_CHANNEL_TYPE_UID_UV_INDEX, "UV Index", CoreItemFactory.NUMBER)
+            .withDescription("Current UV Index")
+            .withStateDescriptionFragment(StateDescriptionFragmentBuilder.create().withReadOnly(true)
+                    .withPattern("%.0f/16").withMinimum(BigDecimal.ZERO).withMaximum(new BigDecimal(16)).build())
+            .withTags(List.of("Status", "Ultraviolet")).build();
```
Comment (lolodomo) [inline bundles/org.openhab.core.thing/src/main/java/org/openhab/core/thing/DefaultSystemChannelTypeProvider.java:409]: Probably rather Measurement than Status ?

File: bundles/org.openhab.core.thing/src/main/java/org/openhab/core/thing/DefaultSystemChannelTypeProvider.java [hunk 2]
```diff
@@ -397,14 +398,24 @@ public class DefaultSystemChannelTypeProvider implements ChannelTypeProvider {
                     StateDescriptionFragmentBuilder.create().withReadOnly(true).withPattern("%.1f %unit%").build())
             .withTags(List.of("Measurement", "Energy")).build();
 
+    /**
+     * UV Index: system wide {@link ChannelType} which shows the UV Index
+     */
+    public static final ChannelType SYSTEM_UV_INDEX = ChannelTypeBuilder
+            .state(SYSTEM_CHANNEL_TYPE_UID_UV_INDEX, "UV Index", CoreItemFactory.NUMBER)
+            .withDescription("Current UV Index")
```
Comment (lolodomo) [inline bundles/org.openhab.core.thing/src/main/java/org/openhab/core/thing/DefaultSystemChannelTypeProvider.java:406]: Index in full lowercase in the description:: Current UV index

**Review by lolodomo — CHANGES_REQUESTED** [review]

File: bundles/org.openhab.core.thing/src/main/java/org/openhab/core/thing/DefaultSystemChannelTypeProvider.java [hunk 3]
```diff
@@ -397,14 +398,24 @@ public class DefaultSystemChannelTypeProvider implements ChannelTypeProvider {
...
+            .withStateDescriptionFragment(StateDescriptionFragmentBuilder.create().withReadOnly(true)
+                    .withPattern("%.0f/16").withMinimum(BigDecimal.ZERO).withMaximum(new BigDecimal(16)).build())
```
Comment (lolodomo) [inline bundles/org.openhab.core.thing/src/main/java/org/openhab/core/thing/DefaultSystemChannelTypeProvider.java:408]: I just read the minimum is 1 and the maximum is 11+. Are you sure about 16 ?

Comment (lolodomo) [reply bundles/org.openhab.core.thing/src/main/java/org/openhab/core/thing/DefaultSystemChannelTypeProvider.java:408]: I believe there is no real max.

Comment (J-N-K) [reply bundles/org.openhab.core.thing/src/main/java/org/openhab/core/thing/DefaultSystemChannelTypeProvider.java:408]: Correct. It's the same as the "Richter scale" for earthquakes.

File: bundles/org.openhab.core.thing/src/main/java/org/openhab/core/thing/DefaultSystemChannelTypeProvider.java (hunk 1 above)
Comment (J-N-K) [reply bundles/org.openhab.core.thing/src/main/java/org/openhab/core/thing/DefaultSystemChannelTypeProvider.java:409]: I agree. If we look at the examples for "Status", it's more a "Measurement".

File: bundles/org.openhab.core.thing/src/main/java/org/openhab/core/thing/DefaultSystemChannelTypeProvider.java (hunk 3 above)
Comment (lolodomo) [reply bundles/org.openhab.core.thing/src/main/java/org/openhab/core/thing/DefaultSystemChannelTypeProvider.java:408]: So you should not define a max and the pattern "x/16" should be changed too.

File: bundles/org.openhab.core.thing/src/main/java/org/openhab/core/thing/DefaultSystemChannelTypeProvider.java (hunk 2 above)
Comment (clinique) [reply bundles/org.openhab.core.thing/src/main/java/org/openhab/core/thing/DefaultSystemChannelTypeProvider.java:406]: Changed

**Review by J-N-K — APPROVED** [review]
LGTM, thanks!

File: bundles/org.openhab.core.thing/src/main/resources/OH-INF/i18n/DefaultSystemChannels.properties [hunk 4]
```diff
@@ -56,3 +56,5 @@ channel-type.system.electric-voltage.label = Electric Voltage
 channel-type.system.electric-voltage.description = Current electric voltage
 channel-type.system.electrical-energy.label = Electrical Energy
 channel-type.system.electrical-energy.description = Current electrical energy
+channel-type.system.uv-index.label = UV Index
+channel-type.system.uv-index.description = Current UV Index
```
Comment (J-N-K) [inline bundles/org.openhab.core.thing/src/main/resources/OH-INF/i18n/DefaultSystemChannels.properties:60]:
```suggestion
channel-type.system.uv-index.description = Current UV index
```

**Comment by J-N-K** [comment]
That's twenty years ago. I don't think we need to support that.

**Comment by magx2** [comment]
@clinique Can you update the [doc](https://www.openhab.org/docs/concepts/units-of-measurement.html#units-of-measurement)

**Comment by clinique** [comment]
@magx2 : [done](https://github.com/openhab/openhab-docs/compare/main...clinique:openhab-docs:patch-3)

**Pull request merged**